file_dbg_macros/src/lib.rs

```diff
@@ -0,0 +1,97 @@
+#[cfg(debug_assertions)]
+use lazy_static::lazy_static;
+#[cfg(debug_assertions)]
+use std::{env, fs::File, sync::Mutex};
+
+// Checks if DBG_FILE_PATH environment variable exists to use as debug file path. If not set,
+// return the OS temp directory joined with default debug filename 'mongodb_odbc.log'
+#[cfg(debug_assertions)]
+fn get_file_path() -> String {
+    if let Ok(file_path) = env::var("DBG_FILE_PATH") {
+        return file_path;
+    }
+    let temp_dir = env::temp_dir();
+    let temp_path = temp_dir.as_path();
+    temp_path.join("mongodb_odbc.log").display().to_string()
+}
+
+#[cfg(debug_assertions)]
+lazy_static! {
+    #[derive(Debug)]
+    pub static ref FILE_PATH: String =  get_file_path();
+
+    pub static ref LOGGER_FILE: Mutex<File> =
+        match std::fs::OpenOptions::new()
+        .write(true)
+        .create(true)
+        .append(true)
+        .open(&*FILE_PATH) {
+                    Err(why) => panic!("couldn't open {:?}: {}", FILE_PATH, why),
```

Comment:
[Suggestion] Maybe this could make the error clear that it simply a file for logging even if we don't have much other files 
```suggestion
                    Err(why) => panic!("couldn't open log file {:?}: {}", FILE_PATH, why),
```